pacemaker.spec.in

```diff
@@ -403,14 +403,26 @@ rm -rf %{buildroot}
 %postun
 %systemd_postun_with_restart pacemaker.service
 
+%pre remote
+systemctl --quiet is-active pacemaker_remote
+if [ $? -eq 0 ] ; then
+    touch %{_localstatedir}/lib/rpm-state/%{name}.restart_pacemaker_remote
```

Comment:
Specfile would perhaps also be wise to use just `rmdir`. And putting initscript into the mix is ugly.